cardano-node/ChangeLog.md

```diff
@@ -24,6 +28,30 @@ None
 
 None
 
+## 1.35.4 -- October 2022
+
+### node changes
+
+- Update plutus to version 1.0.0.1 to enable SECP at protocol version 8
+- Update cardano-crypto-class to version 2.0.0.0.2 to add SECP crypto primitives
```

Comment:
Do we intentionally  have different versioning conventions? x.x.x.x vs x.x.x.x.x

Reply:
The extra . Is useful if you're trying to cut a release in between releases without changing version numbers. This was suggested by @michaelpj

Reply:
Ideally we will stop doing this sort of nonsense soon, though.

Reply:
Yes, that is a CHaP artifact.